lib/hooks.tsx

```diff
@@ -40,6 +40,23 @@ export function useSetter<
   }, [setter, box, value]);
 }
 
+/**
+ * Calls method `setter` on `box` with the arguments `args`, iff `args` is set &
+ * deeply different from before (and the box is alive)
+ */
+export function useSpreadSetter<
```

Comment:
i'm not loving the name. `select` wasnt really a setter to begin with (stretching the usage here) and now it is even less one. maybe just useMethod or something? And then document that it's like `useSetter` except args is an array?